### PR TITLE
skip flaky stashed ops test

### PR DESCRIPTION
**Pull request by wes-carlson**
This is intermittently hitting a check in summarizer that sends an error event when it sees a summaryAck op without seeing the summary. This condition is expected when loading with stashed ops, so this check should be skipped in that scenario at which point this test can be re-enabled.

### CODE REVIEW

File: packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts [hunk 1]
```diff
@@ -805,7 +805,7 @@ describeNoCompat("stashed ops", (getTestObjectProvider) => {
         assert.strictEqual(map3.get(testKey2), testValue);
     });
 
-    it("works with summary while offline", async function() {
+    it.skip("works with summary while offline", async function() {
```
Comment (andre4i) [inline packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts:808]: Can you open an issue and leave a todo so we don't lose track of it?

**Review by andre4i — APPROVED** [review]

**Comment by agarwal-navin** [comment]
> This is intermittently hitting a check in summarizer that sends an error event when it sees a summaryAck op without seeing the summary. This condition is expected when loading with stashed ops, so this check should be skipped in that scenario at which point this test can be re-enabled.

Is there an issue that talks about this? I would like to know more about why this is expected and what the proposed fix is.

**Comment by wes-carlson** [comment]
I put some detail in #10729. It's ultimately caused by SummaryCollection using DeltaManager.on("op") instead of ContainerRuntime.process()

**Review by markfields — APPROVED** [review]
🎂

**Pull request merged**